src/reducers/forms.js

```diff
@@ -0,0 +1,43 @@
+import * as types from '../actions/types';
+
+const initialState = {
+  editingDisabled: false,
+  structureRetrieved: false,
+  waveformRetrieved: false
+};
+
+const forms = (state = initialState, action) => {
+  switch (action.type) {
+    case types.IS_EDITING_TIMESPAN:
+      if (action.code === 0) {
+        return Object.assign({}, state, {
+          editingDisabled: true
+        });
+      }
+
+      return Object.assign({}, state, {
+        editingDisabled: false
+      });
+
```

Comment:
Might need a `break` in this case block?

Reply:
Yeah, when I add a `break` there, it gives a warning in the console, `Line 21:  Unreachable code  no-unreachable`. Maybe because, we have a default `return` statement for that case? And according to the internet this comes from `ESLint` configuration, which I have in my VSCode?

When I remove the `break` statements for the other 2 cases, I got a warning in the console, `Expected a 'break' statement before 'case'  no-fallthrough`, but for this I don't get that.

Reply:
Ah, true because it returns before hitting the `break`.    I think it's complaining about the other 2 scenarios with if there was no `break` as the return may never hit.   Never mind, looks good!